### PR TITLE
OD-493 [Feature] New messages format using file picker

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-493 https://weboo.atlassian.net/browse/OD-493
## Description
Added fix for removing item when click 'Undo'
## Screenshots/screencasts
https://screencast-o-matic.com/watch/crfOI1VerHt
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @ivandevupp @YaroslavOvdii

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -409,6 +409,8 @@ function displayDeletionConfirmation(type, id) {
     }).then(function(deleteResult) {
       if (!deleteResult) {
         restoreItem(sourceType, id);
+
+        return Promise.reject();
```
Comment (squallstar) [inline js/interface.js:413]: Why no error message?

Comment (squallstar) [reply js/interface.js:413]: I don't get it. If you're not handling the promise failure why do you need to reject in first place?

Comment (AndrRyaz) [reply js/interface.js:413]: When we click Confirm and Undo buttons always we will come to `then` in right side of a screenshot thats why i used `Promise.reject` if deleteResult is false which equals when we clicking Undo butto

**Review by squallstar — APPROVED** [review]
@sofiiakvasnevska this is ready to be deployed on **staging** for QA.

- **Risk**: Low
- **Affected**: Studio users using the file picker
- **Verify**: Verify the file picker works as expected

**Pull request merged**